src/content/tutorials/observability/alerting/create-rules/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+linkTitle: Creating alerting rules
+title: How to create alerts and recording rules
+description: Guide explaining how to create alerts and recording rules in the Observability Platform.
+menu:
+  principal:
+    identifier: tutorials-observability-alerting-alerts
+    parent: tutorials-observability-alerting
+weight: 40
+last_review_date: 2025-04-24
+user_questions:
+  - How to create alerts?
+  - How to create recording rules?
+  - What is log-based alerting?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+---
+
+The Giant Swarm Observability Platform provides an [alerting pipeline]({{< relref "/overview/observability/alerting/" >}}) that [you can configure per tenant]({{< relref "/tutorials/observability/alerting/configure-alertmanager" >}}) as well as allow you to create your own alerting and recording rules per tenant.
+
+As Giant Swarm embraces GitOps, alerting and recording rules needs to be defined via [Prometheus Operator](https://prometheus-operator.dev/) PrometheusRules. Those rules can be defined in both your management cluster and workload clusters alike.
+
+**Warning:** As our multi-tenancy aligns tenants across our platform on Grafana Organizations please make sure that the `observability.giantswarm.io/tenant` label defined on your rules references an existing tenant defined in a Grafana Organization. Any PrometheusRules that references an non-existing tenant will be ignored. Learn more about our multi-tenancy in [Multi-tenancy in the observability platform]({{< relref "/tutorials/observability/multi-tenancy/" >}})
+
+## How to define a recording rule
```

Comment:
Isn't like an alerting rule more often used than a recording rule? Shouldn't we be doing alerting rule, then recording rule? idk

Reply:
Maybe, I ordered them that way because recording rules are easier to understand

Reply:
Order is switched